data/gui/window/logging.cfg

```diff
@@ -0,0 +1,192 @@
+#textdomain wesnoth-lib
+###
+### Definition of the window to select logging options.
+### These first five things match the names from logging.cpp
+###
+#define _NAME
+"title_name"
+#enddef
+#define _DEBUG
+"debug"
+#enddef
+#define _INFO
+"info"
+#enddef
+#define _WARN
+"warn"
+#enddef
+#define _ERR
+"err"
+#enddef
```

Comment:
I have no real problem with them being here, whether they are necessary or not, but I do think you should put the enddef on the same line, eg:

```wml
#define _WARN
"warn"#enddef
```